src/libstd/sys/mod.rs

```diff
@@ -80,6 +81,7 @@ cfg_if! {
 cfg_if! {
     if #[cfg(windows)] {
         // On windows we'll just be documenting what's already available
+        #[allow(missing_docs)]
```

Comment:
Rust docs fails to build on the given target because of `#[deny(missing_docs)]` I assumed (because I had the issue when updating rustdoc in the first place). A hidden failure in short. :)